### PR TITLE
auto labeled tag design & replacing between text with draw region

### PR DESCRIPTION
**Pull request by starain-pactera**
1. Change auto labeled tag icon
2. Change workflow of replacing between text with draw region
image
image
image
image

### CODE REVIEW

File: src/react/components/pages/editorPage/editorPage.tsx [hunk 1]
```diff
@@ -742,11 +744,14 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
                         const asset = this.state.assets.find((asset) => asset.id === assetId);
                         if (asset && (asset.state === AssetState.NotVisited || runForAll)) {
                             try {
-                                this.updateAssetState({ id: asset.id, isRunningOCR: true });
+                                this.updateAssetState({id: asset.id, isRunningOCR: true});
                                 await ocrService.getRecognizedText(asset.path, asset.name, asset.mimeType, undefined, runForAll);
-                                this.updateAssetState({ id: asset.id, isRunningOCR: false, assetState: AssetState.Visited });
+                                // await this.props.actions.loadAssetMetadata(this.props.project, asset);
+                                // this.onAssetMetadataChanged(assetMetadata);
+                                this.props.actions.refreshAsset(this.props.project, asset.name);
```
Comment (stew-ro) [inline src/react/components/pages/editorPage/editorPage.tsx:751]: We shouldn't need to introduce a new project action to accomplish this... this was existing behavior of updateAssetState. Can we keep the change as simple as possible without breaking existing behavior?

**Review by stew-ro — APPROVED** [review]

File: src/common/mockFactory.ts [hunk 2]
```diff
@@ -438,10 +442,10 @@ export default class MockFactory {
...
             },
             points: [
-                { x: origin.x, y: origin.y }, // Top left
-                { x: origin.x + size.width, y: origin.y }, // Top Right
-                { x: origin.x, y: origin.y + size.height }, // Bottom Left
-                { x: origin.x + size.width, y: origin.y + size.height }, // Bottom Right
+                {x: origin.x, y: origin.y}, // Top left
+                {x: origin.x + size.width, y: origin.y}, // Top Right
+                {x: origin.x, y: origin.y + size.height}, // Bottom Left
+                {x: origin.x + size.width, y: origin.y + size.height}, // Bottom Right
```
Comment (XinxingLiu) [inline src/common/mockFactory.ts:448]: Could you make sure we don't put unnecessary changes in this PR? Intended styling change should go to another PR.

File: src/common/mockFactory.ts [hunk 3]
```diff
@@ -7,20 +7,20 @@ import {
...
-import { appInfo } from "./appInfo";
-import { IKeyboardBindingProps } from "../react/components/common/keyboardBinding/keyboardBinding";
-import { KeyEventType } from "../react/components/common/keyboardManager/keyboardManager";
-import { IKeyboardRegistrations } from "../react/components/common/keyboardManager/keyboardRegistrationManager";
+import {generateKey} from "./crypto";
+import {randomIntInRange, encodeFileURI} from "./utils";
+import {appInfo} from "./appInfo";
+import {IKeyboardBindingProps} from "../react/components/common/keyboardBinding/keyboardBinding";
+import {KeyEventType} from "../react/components/common/keyboardManager/keyboardManager";
+import {IKeyboardRegistrations} from "../react/components/common/keyboardManager/keyboardRegistrationManager";
```
Comment (XinxingLiu) [inline src/common/mockFactory.ts:23]: Revert unnecessary code style changes.

File: src/common/mockFactory.ts [hunk 4]
```diff
@@ -164,14 +164,14 @@ export default class MockFactory {
...
-        return { containerItems: result };
+        return {containerItems: result};
     }
 
     /**
      * Creates fake IAzureCloudStorageOptions
      */
     public static createAzureOptions(): IAzureCloudStorageOptions {
-        return { sas: "sas" };
+        return {sas: "sas"};
```
Comment (XinxingLiu) [inline src/common/mockFactory.ts:174]: Same here.

**Comment by alex-krasn** [comment]
State does not updates correctly. 2. I think the "batch size" should be dynamic. Now it's confusing -  when app allows user to run auto-labeling for next 10 docs, if there are no that many in the project.
image
image

File: src/common/mockFactory.ts (hunk 4 above)
Comment (starain-pactera) [reply src/common/mockFactory.ts:174]: Got it and we will discuss the criteria of code style. Thanks

**Comment by starain-pactera** [comment]
> State does not updates correctly. 2. I think the "batch size" should be dynamic. Now it's confusing - when app allows user to run auto-labeling for next 10 docs, if there are no that many in the project.
> image
> image

Hello Alex,
This is the max batch size and all documents will be in progress if current batch size number is large than documents' count.
Thanks

**Review by XinxingLiu — DISMISSED** [review]

**Review by yongbing-chen — APPROVED** [review]
